umm-spec-lib/test/cmr/umm_spec/test/generate_and_parse.clj

```diff
@@ -28,19 +28,18 @@
 
 (def tested-collection-formats
   "Seq of formats to use in round-trip conversion and XML validation tests."
-  [:echo10])
+  [:dif10])
```

Comment:
why echo is now dif?

Reply:
I was debating If we should do one at a time and put all together in the end....  but it doesn't really make any difference, once each individual one is working with UMM-C round-trip, they should work together too. So I added the echo10 back.